src/components/bcros/businessDetails/Links.vue

```diff
@@ -126,8 +135,26 @@ const downloadBusinessSummary = async (): Promise<void> => {
 
 /** Creates a draft filing and navigates to the Create UI to file a company dissolution filing. */
 const dissolveBusiness = async (): Promise<void> => {
-  // To be implemented in ticket #23467
-  await new Promise<void>((resolve) => { resolve() })
+  const payload = {
+    custodialOffice: currentBusinessAddresses.value?.registeredOffice,
+    dissolutionType: 'voluntary'
```

Comment:
we can use `FilingSubTypeE.DISSOLUTION_VOLUNTARY` here

Reply:
Changed, thanks